prboom2/src/p_pspr.c

```diff
@@ -2228,19 +2293,19 @@ void P_UpdateBeak(player_t * player, pspdef_t * psp)
 dboolean Heretic_P_CheckAmmo(player_t * player)
 {
     ammotype_t ammo;
-    int *ammoUse;
+    weaponinfo_t *weaponinfo;
```

Comment:
Let's not call that `weaponinfo` - I think that's a recipe for future chaos when the global has the same name :smile: 
Maybe `checkweaponinfo` :shrug:

Reply:
Good callout -- will fix.

Reply:
Aaaand, done.